demo/src/main/java/com/orange/ods/demo/ui/components/appbars/top/AppBarTopCustomizationState.kt

```diff
@@ -0,0 +1,46 @@
+/*
+ *
+ *  Copyright 2021 Orange
+ *
+ *  Use of this source code is governed by an MIT-style
+ *  license that can be found in the LICENSE file or at
+ *  https://opensource.org/licenses/MIT.
+ * /
+ */
+
+package com.orange.ods.demo.ui.components.appbars.top
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.saveable.rememberSaveable
+
+@Composable
+fun rememberAppBarTopCustomizationState(
+    navigationIconEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(true) },
+    actionCount: MutableState<Int> = rememberSaveable { mutableStateOf(1) },
+    overflowMenuEnabled: MutableState<Boolean> = rememberSaveable { mutableStateOf(false) },
```

Comment:
Same question as above, should we use values from `TopAppBarConfiguration.defaultConfiguration` as default values ?